CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Features
 ### Enhancements
 ### Bug Fixes
+- Fix function names and comments in the gradle file for BWC tests ([#795](https://github.com/opensearch-project/neural-search/pull/795/files))
```

Comment:
Could you please provide more details about why we're zip it instead of unzip? For my previous understanding, we fetch the zip archieve from the CI website and unzip it to build the cluster.

Reply:
These functions are actually doing zipping instead of unzipping. This PR does not change any code logic and just fix comments and function names to avoid confusion.

Reply:
Yes, my expression maybe not clear enough. My question is, how we're doing the zip? Could you please describe the whole execution process of the zip* or unzip* function?

Reply:
This is used to install plugin with zip, you can check the gradle file in restart upgrade and rolling upgrade.
```
versions = [ext.neural_search_bwc_version, opensearch_version]
plugin(project.tasks.unZipBwcMlCommonsPlugin.archiveFile)
plugin(project.tasks.unZipBwcKnnPlugin.archiveFile)
plugin(project.tasks.unZipBwcPlugin.archiveFile)
```

Reply:
Here are the steps:
1. pullOpensearchArtifact downloads the whole opensearch artifact with plugins.
2. Tasks like pullBwcPlugin copy the plugin directory within the opensearch artifact plugin folder.
3. Tasks like zipBwcPlugin zips the plugin directory into a zip file.
4. testClusters in restart upgrade and rolling upgrade installs the plugin.

Reply:
makes sense